src/styles/blocks/index.scss

```diff
@@ -0,0 +1,10 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  height: 80vmin;
```

Comment:
its mean size of the clock, not body)
![image](https://github.com/mate-academy/layout_stop-watch/assets/96265844/930abd0e-cebc-4536-a3cd-b1380713e4ed)

Reply:
ah ok sry
ty!